<launch>

  <!-- Rapp core agent nodes -->
  
  <node pkg="rapp_ros_naoqi_wrappings" name="acore_communication_server" type="acore_communication_server.py" output="screen"/>
  <include file="$(find rapp_ros_naoqi_wrappings)/launch/navigation/full_navigation_agent.launch"/>

  
</launch>
